test/unit/xml_description_test.py

```diff
@@ -27,35 +47,49 @@ def test_load_schema_import_error(self, mock_xslt, mock_relax):
         mock_relax.side_effect = KiwiSchemaImportError(
             'ImportError'
         )
-        self.description.load()
+        self.description_from_file.load()
 
     @raises(KiwiValidationError)
     @patch('lxml.etree.RelaxNG')
     @patch('lxml.etree.parse')
     @patch.object(XMLDescription, '_xsltproc')
-    def test_load_schema_validation_error(
+    def test_load_schema_validation_error_from_file(
         self, mock_xslt, mock_parse, mock_relax
     ):
```

Comment:
This is compilable but I would suggest not to format it that way due to [indentation as per PEP8](https://www.python.org/dev/peps/pep-0008/#indentation). I would offer several ways.

As a member of Emacs religion:
```python
    def test_load_schema_validation_error_from_file(self, mock_xslt,
                                                    mock_parse, mock_relax):
    # Code here
```
Or:
```python
def test_load_schema_validation_error_from_file(
        self, mock_xslt, mock_parse, mock_relax):
    # Code here
```
Or:
```python
def test_load_schema_validation_error_from_file(self,
                                                mock_xslt,
                                                mock_parse,
                                                mock_relax):
    # Code here
```
In all cases the code starts from the left side from `):` thing.